lib/xandra/retry_strategy.ex

```diff
@@ -16,18 +16,26 @@ defmodule Xandra.RetryStrategy do
   When a query fails and a retry strategy module was passed as an option, Xandra
   will:
 
-    1. invoke the `c:new/1` callback with the options passed to the failing
-       function to initialize the given retry strategy
+    1. invoke the `c:new/1` callback with options passed to the failing function
+       to initialize the given retry strategy
```

Comment:
I like the previous phrasing better 😬 

```suggestion
    1. invoke the `c:new/1` callback with options passed to the failing function
       to initialize the given retry strategy
```